generator/autogenlist.ts

```diff
@@ -33,6 +33,49 @@ const disabledProviders: AutoGenConfig[] = [
         namespace: 'Microsoft.ContainerService',
         disabledForAutogen: true,
     },
+    {
+        //Disabled until errors are fixed
+        //'datamanagerforagriculturesolutionproperties.properties.openapispecsdictionary.additionalproperties' - TypeError: Cannot convert undefined or null to object
+        basePath: 'agrifood/resource-manager',
+        namespace: 'Microsoft.AgFoodPlatform',
+        disabledForAutogen: true,
+    },
+    {
+        //Disabled until errors are fixed
+        //Enum 'nextPartitionKey' cannot have a value ' ' that result in an empty name. Use x-ms-enum.values to specify the name of the values.
+        //Enum 'nextRowKey' cannot have a value ' ' that result in an empty name. Use x-ms-enum.values to specify the name of the values.
+        basePath: 'adhybridhealthservice/resource-manager',
```

Comment:
Could you remove these from the enabled list below? We shouldn't have duplicates.

Reply:
Ah yes. Will do